CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ Here is a template for new release sections
 - Default values for energy carrier "Heat" for `DEFAULT_WEIGHTS_ENERGY_CARRIERS` with `{UNIT: "KWh_eleq/kWh_therm", VALUE: 1}`. This is still TBD, as there is no source for this ratio yet (#525)
 - Default unit for energy carriers defined in `DEFAULT_WEIGHTS_ENERGY_CARRIERS`: ENERGY_CARRIER_UNIT. Might be used to define the units of flows and LCOE. (#525)
 - New constant variables: TIMESERIES_TOTAL, TIMESERIES_AVERAGE, LOGFILE, RENEWABLE_SHARE, TOTAL_DEMAND, SUFFIX_ELECTRICITY_EQUIVALENT, ATTRIBUTED_COSTS, LCOeleq, DEGREE_OF_SECTOR_COUPLING (#525)
+- Benchmark test to test heat sector components and energy price as time series (#524)
```

Comment:
```suggestion
- Benchmark test for a sector-coupled energy system, including electricity and heat, with a heat pump and an energy price as time series (#524)
```
Can you add a sentence [here](https://github.com/rl-institut/mvs_eland/blob/dev/docs/simulating_with_the_mvs.rst#adding-a-timeseries-for-a-parameter) that this benchmark test provides examplary data for adding a timeseries for a parameter?